src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -267,6 +269,7 @@ protected virtual void Initialize()
                     throw new FormatException(string.Format("Unable to parse {0} file.", ScriptConstants.HostMetadataFileName), ex);
                 }
 
+                ScriptConfig.HostConfig.HostConfigMetadata = hostConfig;
```

Comment:
You've added a "HostConfigMetadata" property to the core SDK JobHostConfiguration? Hopefully we can find a better way. We don't want all our traditional users seeing that, since it doesn't apply to them at all.